endpoints/getting-started-grpc/Dockerfile

```diff
@@ -0,0 +1,28 @@
+# The Google Cloud Platform Python runtime is based on Debian Jessie
+# You can read more about the runtime at:
+#   https://github.com/GoogleCloudPlatform/python-runtime
+FROM gcr.io/google_appengine/python
+
+# Create a virtualenv for dependencies. This isolates these packages from
+# system-level packages.
+RUN virtualenv /env
+
+# Setting these environment variables are the same as running
+# source /env/bin/activate.
+ENV VIRTUAL_ENV -p python3.5 /env
+ENV PATH /env/bin:$PATH
+
+ADD . /hello/
+
+WORKDIR /hello
+
+RUN pip install -r requirements.txt
+
+EXPOSE 8000
+
+# Copy Server
+COPY . /hello/
+
```

Comment:
In general, dont use `ADD`. Should look something like:

```
COPY requirements.txt /tmp/
RUN pip install --requirement /tmp/requirements.txt
COPY . /tmp/
```
see: https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#add-or-copy